.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,25 +1,47 @@
 ---
 name: Bug Report
 about: Something doesn't work correctly in Atmosphère.
-
+#assignees:
 ---
 
 ## Bug Report
 
+[ If any section does not apply, use nil. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
+
 ### What's the issue you encountered?
 
+[ Describe the issue in detail and what you were doing beforehand. ]  
+[ Did you make any changes related to Atmosphère itself? ]  
+[ If so make sure to include details relating to what exactly you changed. ]
+
 ### How can the issue be reproduced?
 
-### Crash report?
+[ Include a detailed step by step process for recreating your issue. ]
+[ Note: If software you have installed can be used to make 'backups' of game files or distributed explicitly for piracy and are connected to why Atmosphère crashed, we will not provide support related to this issue. ]
+
+### Crash Report
 
-(If a crash report was created under /atmosphere/crash_reports/, please upload it to 
-[gist](https://gist.github.com/) and paste the link here.)
+[ Crash reports can be found under ``/atmosphere/crash_reports``. ]  
+[ If your issue caused Atmosphère to crash, include the crash report(s) by creating a [gist](https://gist.github.com/) which you should the link here.
+[ Failure to include a crash report for crash related issues could mean your issue is closed with no notification or reason. ]
```

Comment:
I'd remove this line entirely. If someone posts an issue without a crash report, we'll just ask them for the crash report. No sense in closing the issue there. Maybe put a label on it to indicate that we're waiting on a crash report, but I don't think we have a high enough volume of issues for that to be a realistic concern.

Reply:
I will be revising and removing the line saying that failure to include a crash report will cause your issue to be closed, but I will maintain the fact that the user needs to include a crash report.